spec/workspace-spec.coffee

```diff
@@ -878,6 +878,24 @@ describe "Workspace", ->
           runs ->
             expect(resultPaths.sort()).toEqual([file1, file2].sort())
 
+        describe "when an inclusion path starts with either a tilde or exclamation mark", ->
+          it "the path is used as part of the exclusion array, instead of inclusion", ->
```

Comment:
Agreed - it did feel like a bit of a ramble by the time I finished writing it. I'll update it.